src/Middleware/CacheMiddleware.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Dumbo\Middleware;
+
+use Dumbo\Context;
+use Psr\Http\Message\ResponseInterface;
+
+class CacheMiddleware
```

Comment:
I think I made a mistake early on regards to naming everything Helpers, so calling this `Middleware` stands out.

I'd be happy to keep this as `Middleware`, if we make an effort (in another PR) to separate Helpers and Middleware.